### PR TITLE
Total debt total saving graph

### PR DESCRIPTION
**Pull request by anthonySchafrik**
Adds total debt total and total saving graph
Fixed horizontal bars so it starts at 0 along with cleaning up the code some in TopSpendingGraphs

### CODE REVIEW

File: src/components/SnapshotResults.jsx [hunk 1]
```diff
@@ -50,15 +98,19 @@ class SnapshotResults extends React.Component {
...
         </div>
-        <div>
-          <h3>Debt to Savings Ratio</h3>
+        <div style={{ width: "30%", height: "30%" }}>
+          <HorizontalBar
+            data={DebtSavingGraph}
+            options={DebtSavingGraphOptions}
+          />
+          {/* <h3>Debt to Savings Ratio</h3>
           <span>
```
Comment (deduced) [inline src/components/SnapshotResults.jsx]: Let's remove this comment if it's not being used.

File: src/components/TopSpendingGraphs.jsx [hunk 2]
```diff
@@ -1,27 +1,21 @@
 import React from "react";
-// import { HorizontalBar } from "react-chartjs-2";
-import { Bar } from "react-chartjs-2";
+import { HorizontalBar } from "react-chartjs-2";
+// import { Bar } from "react-chartjs-2";
```
Comment (deduced) [inline src/components/TopSpendingGraphs.jsx]: Remove this comment if not being used.

File: src/components/SnapshotResults.jsx (hunk 1 above)
Comment (anthonySchafrik) [reply src/components/SnapshotResults.jsx]: Removed

File: src/components/TopSpendingGraphs.jsx (hunk 2 above)
Comment (anthonySchafrik) [reply src/components/TopSpendingGraphs.jsx]: Removed

**Pull request merged**